docs/src/gs/install-cluster-api.md

```diff
@@ -70,18 +55,18 @@ allow dynamic-group [your dynamic group name] to manage load-balancers in compar
 Initialize management cluster and install CAPOCI
 
    ```bash
-      clusterctl init --infrastructure oci
+      clusterctl init --infrastructure oci:v0.6.0
```

Comment:
NIT: should we use an ENV like we do for CCM instead? and have the user look up the current version? Otherwise we have to remember to bump this version number on each release. 🤷

Reply:
yeah, I wanted to fix this quickly for now, will update it as per CCM